girder_worker/__main__.py

```diff
@@ -1,11 +1,36 @@
+import pkg_resources as pr
+from . import config
 from .app import app
 
+def get_plugin_tasks(ep):
+    try:
+        plugin_class = ep.load()
+    except ImportError:
+        app.log.warn('Could not load \'%s\', skipping.' % str(ep))
+
+    plugin = plugin_class(app)
+
+    return plugin.task_imports()
+
 
 def main():
+    includes = []
+    for ep in pr.iter_entry_points(group='girder_worker_tasks'):
+        # If this is the girder_worker EntryPoint
+        if ep.module_name == __package__:
+            # And core_tasks config is True
+            if config.getboolean('girder_worker', 'core_tasks'):
```

Comment:
What if the `core_tasks` option does not exist? I think defaulting to True would be slightly better than either raising an error or defaulting to False.